core/embed/trezorhal/unix/common.c

```diff
@@ -28,14 +28,16 @@
 #include "display.h"
 #include "memzero.h"
 
+extern void __attribute__((noreturn)) main_clean_exit(int);
```

Comment:
This `extern` indicates that there's an issue with the dependency order. However, let's leave it as is for now, since we are gradually moving towards not having `common.c`/`common.h` at all, which could resolve this issue automatically.

Reply:
`build_bootloader_emu` is failing because of this

Reply:
extern removed and bootloader_emu build fixed in 5475558ff22795b24ec380d4bc5b2ce6900251ab